content/docs/studio/get-started.md

```diff
@@ -117,7 +117,8 @@ You should now see that a project has been added in your dashboard.
 
 2. You can [add new models](/doc/studio/user-guide/model-registry/add-a-model)
    to the model registry by clicking on the `Add a model` button and following
-   on-screen instructions.
+   on-screen instructions. Both remote and local models can be imported this
+   way.
```

Comment:
What does it mean to "import a local model" ? I.e. f it's local why do you need to import it? "Local" means already in the project, right? Or do we mean in the local file system but external to the project?

> I'm thinking along DVC terms here, please correct me if I'm wrong.

Reply:
Yes, local file means a file that is already in the project, but is not registered as a model yet. So, it wouldn't appear in the model registry until you register it (add it as model) first.

Reply:
I see. Term "import" here is confusing. Maybe generalize to "register" or "import and/or register" or something like that?

Reply:
Also, when we say import do we mean `dvc import(-url)` ? I.e. does it save a connection to the source for later `dvc update`? Otherwise maybe just call it "download/fetch/transfer into"

Reply:
Updated.

Reply:
Thanks